gitlint/tests/samples/commit_message/sample1

```diff
@@ -4,3 +4,6 @@ This is the first line of the commit message body and it is meant to test a line
 This line has a tråiling space. 
 This line has a trailing tab.	
 # This is a cömmented  line
+# ------------------------ >8 ------------------------
```

Comment:
I think we should add some non-commented text after the cutline to properly test this use-case since commented lines are being ignored regardless.